stargate-core/app/com/salesforce/mce/stargate/utils/AppConfig.scala

```diff
@@ -0,0 +1,11 @@
+package com.salesforce.mce.stargate.utils
+
+import com.typesafe.config.ConfigFactory
+import play.api.Configuration
+
+object AppConfig {
```

Comment:
Don't think we need to do the config retrieval and parsing ourselves, it should be already available in `play.api.http.SessionConfiguration` https://www.playframework.com/documentation/2.8.x/api/scala/play/api/http/SessionConfiguration.html#cookieName:String

Reply:
Thanks for pointing this out. I've updated the code. Pl check